### ISSUE TITLE
Test WPML

### ISSUE THREAD

**Issue by jonathanbossenger** [WordPress/Learn#2338]

**Comment by unprintedch** [comment]
Here is a summary of the first results :

Here are some of the wpml capabilites

- [x] Language selector does not show as country flags. (Many countries speak multiple languages, so describing a language with a flag should be avoided.)
- [x]  Languages can be changed without needing to log in to one's WordPress.org account.
- [x]  Once a language is selected, it is applied globally.
- [x]  Content of the selected language is shown above non-translated content.
Would need some custom (simple) dev
- [x]  English content is still shown where translations don't exist.
- [x]  Translators do not have permission to create or publish new content.
- [x]  Translators have permission to both translate any content, and review content translated by others.
- [x]  A flow can be implemented where translations must be approved (reviewed) before published.
- [ ]  Sensei content types can be translated in the same way as native post types.
Need to be deeply tested
- [x]  Taxonomies can be translated. 
- [ ]  Translated content is indexed and searchable by Jetpack Search.
Need to be deeply tested

In short, the fact is that WPML is rather complete in terms of functionality.
BUT! It adds much complexity and is sometime capricious, prompt to bugs and leads to frustration.

There is no (in our experience) other real viable alternative except polylang which is not as complete in terms of features.

**Comment by jonathanbossenger** [comment]
Closing this issue as we've decided to stick [to GPL compliant plugins](https://github.com/WordPress/Learn/issues/2231#issuecomment-2024198287).
